iis-wf/iis-wf-referenceextraction/src/test/resources/eu/dnetlib/iis/wf/referenceextraction/patent/data/retriever/report_from_cache.json

```diff
@@ -1,3 +1,3 @@
-{"key":"processing.referenceExtraction.patent.retrieval.fromCache.total", "type":"COUNTER", "value": "2"}
+{"key":"processing.referenceExtraction.patent.retrieval.fromCache.total", "type":"COUNTER", "value": "1"}
```

Comment:
I think the similar remark is applicable here: if we had one successfully processed record and one failure stored in cache then I would expect to see `2` among `fromCache.total` because in fact we retrieved information about 2 records from cache. 

This case is definitely more debatable than the first one so we can discuss what would be more meaningful. Although again, this should not be changed only because we stopped storing empty records in cache. We could change the semantics behind `fromCache.total` but independently from the core change of this pull request.